package.json

```diff
@@ -36,34 +36,51 @@
               "id": "walkthrough",
               "title": "Welcome to AKS Extesion",
               "description": "Azure Kubernetes Service (AKS) Extension for Visual Studio Code helps enable AKS developers with one-click quick to deploy features with in vscode eco-system.",
+              "media": {
+                "image": "./docs/book/src/features/features.md",
+                "altText": "AKS Extension"
+              },
               "steps": [
+                {
+                    "id": "summarycommand",
+                    "title": "All Features",
+                    "description": "Once you successfully log in with your Azure Account, you can view all AKS clusters in your Azure subscriptions(s) under the section named **Azure**. You can right click on your AKS cluster and click a menu item to perform following actions.",
+                    "media": { 
+                      "markdown": "./docs/book/src/features/features.md"
+                    }
+                },
                 {
                   "id": "runcommand",
                   "title": "Cluster Operation Command",
-                  "description": "This step will run a command and check off once it has been run.\n[Run Command](command:getting-started-sample.runCommand)",
-                  "media": { "image": "media/image.png", "altText": "Empty image" },
-                  "completionEvents": ["onCommand:getting-started-sample.runCommand"]
+                  "description": "AKS Managed Cluster Operations supported by this extension, like create cluster, delete, start or stop cluster from show properties page, rotate cluster certificate, etc.",
```

Comment:
Instead of the word "like", can we use "such as" 

"such as create/delete cluster, start/stop cluster, show cluster properties, rotate cluster certificates, etc"